lib/WebDriver/Exception/CurlDetails.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace WebDriver\Exception;
+
+
+final class CurlDetails extends \Exception {
```

Comment:
Move to lib/WebDriver/Service/CurlServiceException.php and rename class to CurlServiceException